routes/assignments.js

```diff
@@ -0,0 +1,21 @@
+var express = require('express');
+var router = express.Router();
+
+/* List ALL Assignments, no grades */
+router.route('/assignments')
+  .get();
+
+/* Make Submission Assignment */
+router.route('/assignments')
+  .post();
+
+/* Make Evaluation Assignment */
+router.route('/assignmentsTHISMAKESITNOTDUPLICATE')
+  .post();
```

Comment:
I just copied the paths from the API blueprint, but I can change it in both places.